Source/WebGPU/WGSL/AttributeValidator.cpp

```diff
@@ -189,7 +189,14 @@ void AttributeValidator::visit(AST::Variable& variable)
         if (is<AST::BindingAttribute>(attribute)) {
             if (!isResource)
                 error(attribute.span(), "@binding attribute must only be applied to resource variables");
-            auto bindingValue = downcast<AST::BindingAttribute>(attribute).binding().constantValue()->integerValue();
+            // https://gpuweb.github.io/cts/standalone/?q=webgpu:shader,validation,parse,attribute:expressions:value=%22override%22;attribute=%22binding%22
```

Comment:
these comments could be removed if you prefer, I left them to indicate which test shows this branch is taken